packages/foundation-models/README.md

```diff
@@ -60,26 +60,42 @@ const chatClient = new AzureOpenAiChatClient({
 });
 ```
 
-### Azure OpenAI Client
+### Chat Client
 
-The Azure OpenAI client can then be used to send chat completion or embedding requests to models deployed in the SAP generative AI hub.
+Use the `AzureOpenAiChatClient` to send chat completion requests to an OpenAI model deployed in SAP generative AI hub.
 
-#### Chat Client
+The client sends request with Azure OpenAI API version `2024-06-01`.
+Set a different version together with other request configuration in the `requestConfig` parameter.
```

Comment:
[q] If user sets a newer version, are we sure that it works with our client? 
Can we instead say something like this client is generates using api version spec so and so, and is backward compatible (I imagine it is??).

Reply:
I will remove line 68. It also feels strange to have a different API version in the header, as our client is generated.